Surface.pyx

```diff
@@ -16,32 +16,38 @@ from SurfaceBudget cimport SurfaceBudget
 from NetCDFIO cimport NetCDFIO_Stats
 import cython
 from thermodynamic_functions import exner, cpm
-from thermodynamic_functions cimport cpm_c, pv_c, pd_c, exner_c
+from thermodynamic_functions cimport cpm_c, pv_c, pd_c, exner_c, qv_star_c
```

Comment:
I think it's safe to remove all the thetali stuff in this file, but please double check to make sure.

Reply:
actually in Surface.pyx we should not make changes - here inputs from the papers is given in term of theta (no one measure entropy as far as I know) and is used to compute the MO surface layer properties

Reply:
Oh you are right! I think we can remove some parts of the code that checks if s or thli in PV. I'll add specific comments below.